aql/src/main/java/org/ehrbase/aql/dto/condition/ConditionComparisonOperatorSymbol.java

```diff
@@ -25,7 +25,8 @@ public enum ConditionComparisonOperatorSymbol {
     GT_EQ("greaterOrEqual", ">="),
     GT("greaterThan", ">"),
     LT_EQ("lessOrEqual", "<="),
-    LT("lessThan", "<");
+    LT("lessThan", "<"),
+    LIKE("like", "like");
```

Comment:
It seems that this enum reflects `COMPARABLEOPERATOR` parser rule from the aql grammar. 
This means that the operators will also be available in node predicates, which is not permitted.

Reply:
removed from the enum